flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -686,6 +697,17 @@ public RocksDBNativeMetricOptions getMemoryWatcherOptions() {
 		return options;
 	}
 
+	/**
+	 * Gets the thread number will used for downloading files from DFS when restore.
+	 */
+	public int getNumberOfRestoringThreads() {
+		return restoringThreadNum == -1 ? RocksDBOptions.CHECKPOINT_RESTORE_THREAD_NUM.defaultValue() : restoringThreadNum;
```

Comment:
Can we also rename the field that it matches the getter?

Reply:
and `setrestoringThreadNum` as well then